hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/SnapshotDeletingService.java

```diff
@@ -610,6 +615,37 @@ public void submitRequest(OMRequest omRequest) {
     }
   }
 
+  public static boolean isBlockLocationInfoSame(OmKeyInfo prevKeyInfo,
+                                                OmKeyInfo deletedKeyInfo) {
+    OmKeyLocationInfoGroup deletedOmKeyLocation =
+        deletedKeyInfo.getLatestVersionLocations();
```

Comment:
There is already a check for the size of the locations list below. 
```
    if (deletedLocationList.size() != prevLocationList.size()) {
      return false;
    }
```

Reply:
I guess this is the blockLocationList correct me if I wrong, I wanted to just put another check on the number of versions are also same. Just to be on the safe side so that we don't end up deleting some random data because of some bug in the version maintenance. Say if they start supporting bucket versioning deletion of particular version.

Reply:
Oh, I get it now. Agreed. Will update the patch with necessary changes.